src/monitors/radiationFlux.cpp

```diff
@@ -0,0 +1,181 @@
+#include "radiationFlux.hpp"
+
+ablate::monitors::RadiationFlux::RadiationFlux(std::vector<std::shared_ptr<radiation::Radiation>> radiationIn, std::shared_ptr<domain::Region> radiationFluxRegionIn)
+    : radiation(std::move(radiationIn)), radiationFluxRegion(std::move(radiationFluxRegionIn)) {}
+
+ablate::monitors::RadiationFlux::~RadiationFlux() {}
+
+void ablate::monitors::RadiationFlux::Register(std::shared_ptr<solver::Solver> solverIn) {
+    Monitor::Register(solverIn);
+
+    // update the name
+    name = radiationFluxRegion->GetName() + name;
+
+    DMLabel radiationFluxRegionLabel = nullptr;
+    PetscInt regionValue = 0;
+    domain::Region::GetLabel(radiationFluxRegion, solverIn->GetSubDomain().GetDM(), radiationFluxRegionLabel, regionValue);
+
+    // Now create a sub dm with only the faces
+    DMPlexFilter(solverIn->GetSubDomain().GetDM(), radiationFluxRegionLabel, 1, &fluxDm) >> utilities::PetscUtilities::checkError;
```

Comment:
Instead of using one here, you can use the region value you pulled on line 16.  Most often it is 1, but not always.